code/controllers/subsystems/records.dm

```diff
@@ -243,16 +222,17 @@
 	for(var/datum/record/general/t in records)
 		var/name = sanitize(t.name)
 		var/rank = sanitize(t.rank)
-		var/real_rank = make_list_rank(t.real_rank)
 
 		var/isactive = t.physical_status
 		var/department = 0
 		var/depthead = 0            // Department Heads will be placed at the top of their lists.
 
+		rank = replacetext(rank, "Interim ", "")
+		rank = replacetext(rank, "Acting ", "")
```

Comment:
I'm not sure I agree with this way of doing it. It certainly seems obscure and undocumented in game. They should probably just be jobs in ID card modification. or at the very least, you should be able to tell from that program what to do.

Reply:
This is a pretty shitty way to do this, yes.